packages/kbn-i18n/src/react/provider.tsx

```diff
@@ -39,7 +39,7 @@ export class I18nProvider extends React.PureComponent {
     return (
       <IntlProvider
         locale={i18n.getLocale()}
-        messages={i18n.getMessages()}
+        messages={i18n.getTranslation().messages}
         defaultLocale={i18n.getDefaultLocale()}
         formats={i18n.getFormats()}
```

Comment:
__question:__ btw, should it be `formats={i18n.getTranslation().formats}` and ` defaultFormats={i18n.getFormats()}` or not? @pavel06081991 @LeanidShutau